tasks/main.yml

```diff
@@ -1,15 +1,18 @@
 ---
 - name: OS specific setup
   include: include: "setup-{{ ansible_os_family }}.yml"
+  when: docker_install
```

Comment:
The reason for this change is really for safety and to guard against user error. If you're so inclined you can explicitly state if a host should or should not have docker installed on the system. This prevents an admin from accidentally installing docker on a host that shouldn't have it installed in the first place. So take the following as an example.
```
[webhost]
host1
host2
host3 <== this host shouldn't ever have docker installed.
host4
```

Now, lets say that an admin some time down the road forgets taht `host3` shouldn't have docker installed and puts the webhost group in the playbook because lets say the docker repo needs to be updated across all docker hosts. The role will modify docker on all those hosts but will also install docker on `host3`. This change simply gives the option to have that extra safety there so you can make sure some hosts never get it installed regardless of user error.